v2/cmd/coil-controller/sub/run.go

```diff
@@ -2,12 +2,13 @@ package sub
 
 import (
 	"fmt"
-	v2 "github.com/cybozu-go/coil/v2"
 	"net"
 	"os"
 	"strconv"
 	"time"
 
+	v2 "github.com/cybozu-go/coil/v2"
```

Comment:
I don't mind how the packages are organized into blocks, but at least all the packages from Coil should appear in a same block.

```
v2 "github.com/cybozu-go/coil/v2"
coilv2 "github.com/cybozu-go/coil/v2/api/v2"
"github.com/cybozu-go/coil/v2/pkg/constants"
"github.com/cybozu-go/coil/v2/pkg/indexing"
"github.com/cybozu-go/coil/v2/pkg/ipam"
"github.com/cybozu-go/coil/v2/runners"
```